app/src/main/java/org/thoughtcrime/securesms/conversation/disappearingmessages/DisappearingMessages.kt

```diff
@@ -58,9 +56,9 @@ class DisappearingMessages @Inject constructor(
 
         dangerButton(
                 text = if (message.expiresIn == 0L) R.string.confirm else R.string.set,
-                contentDescription = if (message.expiresIn == 0L) R.string.AccessibilityId_confirm else R.string.AccessibilityId_setButton
+                contentDescriptionRes = if (message.expiresIn == 0L) R.string.AccessibilityId_confirm else R.string.AccessibilityId_setButton
         ) {
-            set(message.threadId, message.recipient.address, message.expiryMode, message.recipient.isClosedGroupRecipient)
+            set(message.threadId, message.recipient.address, message.expiryMode, message.recipient.isClosedGroupV2Recipient)
```

Comment:
Is this right? Shouldn't you be checking for either type of closed groups here? Legacy or v2?